### ISSUE TITLE
is_volume_attachement not avaliable 

### ISSUE THREAD

**Issue by lucap01** [IBM-Cloud/terraform-provider-ibm#1066]
ibm_is_volume can not be used due to a lack of attachment functionality.

The basic attachment function for VPC gen1, gen2 needs to be added.

**Comment by hkantare** [comment]
You can use volume while provision of instance..
https://ibm-cloud.github.io/tf-ibm-docs/v1.1.0/r/is_instance.html
`volumes` attribute can be used to attach volumes.

**Comment by lucap01** [comment]
Yes, volumes block section can be used. Most of the use case in project requires attaching and detaching volumes while vm is running. This basic functional is avaliable from web ui level, so i believe it can be added as attach/detach function in terraform as well in future.

Here example for Power servers works in that way

data "ibm_pi_instance_volumes" "ds_volumes" {
  pi_instance_name     = "volume_1"
  pi_cloud_instance_id = "49fba6c9-23f8-40bc-9899-aca322ee7d5b"
}

**Comment by powellquiring** [comment]
In AWS, sorry a little complicated
```
resource "aws_ebs_volume" "centos" {
  for_each = local.volumes_centos
  availability_zone = data.aws_availability_zones.available.names[0]
  size = each.value.size
  tags = {
    Name = local.name_centos
    key = each.key
    size = each.value.size
    device_name = each.value.device_name
  }
}
resource "aws_volume_attachment" "centos" {
  for_each = local.volumes_centos
  device_name = each.value.device_name
  volume_id   = aws_ebs_volume.centos[each.key].id
  instance_id = aws_instance.centos.id
  # provisioner remote-exec {
}


resource "aws_instance" "centos" {
...
```

The ability to attach/detach volumes is useful.

Now property changes for aws_instance.centos that forces a destroy/re-create of the instance do not destroy the data volumes attached.

**Comment by kavya498** [comment]
Closing this issue as support is available from 1.28.0 and above releases..
Please use latest provider..

Thanks..
